### ISSUE TITLE
Test failure with Python 3.9

### ISSUE THREAD

**Issue by Aniket-Pradhan** [G-Node/python-odml#350]
Hello devs!

We had successfully included python-odml inside Fedora (It is available in the testing repos, and will be available in the stable repos in about a week :clinking_glasses: )

We also did a test build for Fedora 33 (which will have Python 3.9), and some tests fail. :(
The main error faced during the tests is:

```
>       if not isinstance(obj_list, collections.Iterable):
E       AttributeError: module 'collections' has no attribute 'Iterable'
```

As per the docs from Python 3.9:

"The abstract base classes in collections.abc no longer are exposed in the regular collections module. This will help create a clearer distinction between the concrete classes and the abstract base classes."

A detailed build and test log can be found [here](https://copr-be.cloud.fedoraproject.org/results/@python/python3.9/fedora-rawhide-x86_64/01223887-python-odml/build.log.gz)

I can try digging around this as well, but it would take me some time as I am a bit occupied with some other work. Talking about that, there should be no hurry, because there is a lot of time till the stable release of Python 3.9.

**Comment by tirkarthi** [comment]
I have created PR #351 for this. Thanks.

**Comment by mpsonntag** [comment]
Thanks a lot for the issue and the fix!

**Comment by Aniket-Pradhan** [comment]
Just a small update from the Python devs:

"In Python 3.9.0a4 we have managed to revert the change that disables importing
abstract base classes (collections.abc) directly form the regular collections
module. It was reverted to ease transition for projects that did not yet
actually dropped support for Python 2.7. The removal is scheduled for Python
3.10 instead."

**TL;DR**: This patch was supposed to be for Py3.9, but now this patch would effectively work for Py3.10. Py3.9 has reverted the changes.
